backend/api_documentation.md

```diff
@@ -0,0 +1,60 @@
+
+# API Documentation:
+
+#### Full Issue Profile
+`/issues/profile/<issue_id|>`  
+Get an issue's full profile. If no `issue_id` is provided, a relevant issue will be selected and returned.  
+Sample endpoint response:  
+```
+{
+    "issue_title": "An Act promoting climate change adaptation, environmental and natural resource protection, and investment in recreational assets and opportunity",
+    "days_until_vote": 11,
+    "desired_outcome": "yea",
+    "number_of_positive_votes_outstanding": 6,
+    "confidence_rating": 95,
+    "legislators": [
+        {
+            "name": "Ted Deutch",
+            "chamber": "House",
+            "state": "Florida",
+            "district": "22",
+            "party": "Democrat",
+            "id": 1
+        },
+        ...
+    ]
+}
+```
+
+#### Issue Preview
+`/issues/preview/`  
+Returns the previews of current and archived issues.  
+Sample endpoint response:  
+```
+{
+    "current": [
```

Comment:
It's not a huge deal, (since I don't  really care, and I'm the only one who would) but these sample responses are not in [JSONapi format](http://jsonapi.org/examples/). 

But like I said we can go with whatever format you're most comfortable with for now. As long as we have nice & consistent separation of concerns at the end of things.